model/src/main/kotlin/config/ScanStorageConfiguration.kt

```diff
@@ -118,6 +118,11 @@ data class PostgresStorageConfiguration(
      */
     val sslrootcert: String? = null,
 
+    /**
+     * The number of threads to use for the storage dispatcher.
+     */
+    val storageThreads: Int = 5,
```

Comment:
The name "storage threads" was used before, but it is actually incorrect; coroutines are not directly mapped to threads, and the property rather defines the number of transactions spawned in parallel. So, while at it, should it be renamed to something like `parallelTransactions` ?